app/assets/stylesheets/static_pages.css.scss

```diff
@@ -159,6 +159,99 @@ main {
   }
 }
 
+//-------------------------------------------------------------
+// Testimonials Component | Homepage
+//-------------------------------------------------------------
+.testimonials {
+  blockquote {
+    border-left: none;
+    max-width: 91%;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    justify-content: center;
+  }
+
+  p, cite {
+    font-size: px-to-rem(18px);
+  }
+
+  .testimonial {
+    height: inherit;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+  }
+
+  cite {
+    align-self: flex-start;
+  }
+
+  .photo {
+    padding: 0 0 px-to-rem(10px) 0;
+  }
```

Comment:
It looks like that css can be removed, but the class is also being used for the code in the min-width: 48em media query to put space around the photo when the image goes next to the text instead of above it and if I remove it, then the image in the testimonial is too close to the text when the text is next to the image.

Also, I was styling the image directly but then I put it inside a div to fix an issue with Firefox not showing the image at the correct dimensions and I thought it would be better to style the div container instead of the image. 

If I remove this code then the image is too close to the text:
```
    .photo {
      padding: 0 px-to-rem(15px) 0 px-to-rem(10px);
    }
```

However it looks like it would be fine if we got rid of that class and just styled the image tag inside the testimonial within that same media query:
```
    img {
      padding-right: px-to-rem(10px);
    }
```

What do you think?

Reply:
if it breaks the image - we should leave it. lets just update to use the variable and that's it.

Reply:
Thanks @cherchezlafemme I updated it to use the variable.